Documentation/building/unix-instructions.md

```diff
@@ -72,7 +72,3 @@ PDBs aren't generated by Roslyn on Unix. https://github.com/dotnet/roslyn/issues
 Test runs are currently disabled when building on Unix. https://github.com/dotnet/corefx/issues/1776
 
 System.Diagnostics.Debug.Tests does not build on Unix. https://github.com/dotnet/corefx/issues/1609
-
-Mono fails when trying to get custom attributes on CoreFx assemblies. https://bugzilla.xamarin.com/show_bug.cgi?id=29679
```

Comment:
Note that the fix will appear in the upcoming Mono 4.2 release, so it's technically not fixed yet, but we have the workaround in build.sh so nobody should ever experience this. I'm fine with removing this.